src/commands/singleCommand.ml

```diff
@@ -66,6 +68,9 @@ let main all weak debug verbose verbose_indent json profile quiet module_
   let config_root = CommandUtils.guess_root (Some(root)) in
```

Comment:
Ahhh....I think the assert you're hitting is due to this. One of the things that `Path.make` does is `realpath`'s the path. So if you're using symlinks and stuff, maybe this `config_root` is different than the `opt_root` below? Then you're hitting an assert when the config things you're using two different roots?

That being said, I can't seem to repro myself :/